metal/drivers/sifive_i2c0.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2018 SiFive, Inc */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#ifndef METAL__DRIVERS__SIFIVE_I2C0_H
+#define METAL__DRIVERS__SIFIVE_I2C0_H
+
+#include <metal/i2c.h>
+
+struct __metal_driver_vtable_sifive_i2c0 {
+    const struct metal_i2c_vtable i2c;
+};
+
+__METAL_DECLARE_VTABLE(__metal_driver_vtable_sifive_i2c0)
+
+struct __metal_driver_sifive_i2c0 {
+    struct metal_i2c i2c;
+    unsigned int init_done;
```

Comment:
any reason this need to be unsigned?

Reply:
will not hold negative values, MISRA C standard recommends unsigned usage since overflow behavior is defined unlike signed types.

Reply:
Not sure it matter in this case, since init_done is used as an indicator for not done (0) and done (non-0)